sim/side.ts

```diff
@@ -629,19 +629,25 @@ export class Side {
 
 			switch (choiceType) {
 			case 'move':
-				let targetLoc = 0;
+				const original = data;
+				const error = () => this.emitChoiceError(`Conflicting arguments for "move": ${original}`);
+				let targetLoc: number | undefined;
 				let megaOrZ = '';
 				while (true) {
 					if (/\s-?[1-3]$/.test(data)) {
+						if (targetLoc !== undefined) error();
```

Comment:
I might be missing something, but shouldn't these be early ``return error()``; ?

Reply:
these should all be `return`s, yeah.